pkg/node/node.go

```diff
@@ -278,11 +279,25 @@ func NewBee(
 		return nil, fmt.Errorf("compute overlay address: %w", err)
 	}
 
-	if nonceExists && o.TargetNeighborhood != "" {
-		logger.Warning("an overlay has already been created before, skipping targeting the selected neighborhood")
-	}
+	if o.TargetNeighborhood != "" {
+		if nonceExists {
```

Comment:
if the target neighborhood is specified, on each restart, wouldn't the directories would be nuked?